main/src/main/res/layout/map_zoom_control.xml

```diff
@@ -7,52 +7,42 @@
         android:id="@+id/container_followmylocation"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_above="@id/container_zoomin"
+        android:layout_above="@id/container_zoom"
         android:layout_alignParentRight="true"
         android:layout_marginRight="13dp"
         android:layout_marginBottom="6dp">
         <ImageButton
-            android:id="@+id/background_followmylocation"
-            android:tooltipText="@string/menu_centerposition"
-            android:src="@drawable/map_button_background"
-            style="@style/map_quicksettingsbutton" />
-        <ImageView
             android:id="@+id/map_followmylocation_btn"
-            android:tooltipText="@string/menu_centerposition"
-            android:layout_width="24dp"
-            android:layout_height="24dp"
-            android:layout_centerInParent="true"
-            android:contentDescription="@string/menu_centerposition"
-            style="@style/map_quicksettingsbutton" />
+            android:src="@drawable/map_quick_settings"
+            style="@style/map_quicksettingsbutton"
+            android:tooltipText="@string/quick_settings"
```

Comment:
tooltipText and contentDescription should point to "follow my position", currently a long-tap on this icon would show a "quick settings" tooltip